Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.Properties.cs

```diff
@@ -79,6 +91,24 @@ public ExpandDirection ExpandDirection
             set { SetValue(ExpandDirectionProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the Expander control should be in Expand or Overlay mode.
+        /// </summary>
+        public ExpanderDisplayMode DisplayMode
+        {
+            get { return (ExpanderDisplayMode)GetValue(DisplayModeProperty); }
+            set { SetValue(DisplayModeProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the ContentOverlay of the control.
+        /// </summary>
+        public object ContentOverlay
```

Comment:
This should be 
 public UIElement ContentOverlay
        {
            get { return (UIElement)GetValue(ContentOverlayProperty); }
            set { SetValue(ContentOverlayProperty, value); }
        }

Reply:
No worry. I tried with object to see if it worked better.